src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const thumbs = document.getElementById('thumbs');
+
+thumbs.addEventListener('click', (eventt) => {
```

Comment:
The parameter `eventt` in the arrow function is likely a typo. It should be `event` to follow common naming conventions and avoid confusion.